cypress/e2e/check_account_balance_ddt_test.cy.js

```diff
@@ -0,0 +1,59 @@
+import accountsData from "../fixtures/account_balance_data.json";
+import { faker } from "@faker-js/faker";
+import { LoginPage } from "../page-objects/login_page";
+import { UserApi } from "../api/users_api";
+import { HomePage } from "../page-objects/home_page";
+
+describe("create accounts and check accounting balances", () => {
+  let username = faker.internet.userName();
+  let password = faker.internet.password();
+  let email = faker.internet.exampleEmail();
+
+  before(() => {
+    cy.intercept("/tegb/register").as("registration_loading");
+
+    new LoginPage()
+      .openTegBUrl()
+      .clickRegistrationButton()
+      .typeUsername(username)
+      .typePassword(password)
+      .typeEmail(email)
+      .clickRegisterButton();
+
+    cy.wait("@registration_loading");
+  });
+
+  beforeEach(() => {
+    new LoginPage()
+      .openTegBUrl()
+      .typeUsername(username)
+      .typePassword(password)
+      .clickLoginButton();
+  });
+
+  accountsData.forEach((accountData) => {
+    it(`create account no. ${accountData.row + 1}`, () => {
```

Comment:
Ještě by bylo dobré tady zakomentovat, proč nejsou testovány všechny varianty testu nebo úplně nejlepší (a opět nejtěžší) udělat v JSON disabled element s popisem, proč není kombinace testovaná a pak zde udělat if(), který by toto řešil. Toto ale nebylo vůbec vyžadované, jen posílám inspiraci :-)

Mohlo by to vypadat třeba takto:
**JSON:**
```json
  {
    "row": 4,
    "accountName": "Test_Account_",
    "balance": "1234567890.00",
    "disabled": true,
    "disabledReason": "BUG123 - 500 internal server error, in account creation"
  }
```

**Test:**
``` javascript
import accountsData from "../fixtures/account_balance_data.json";

describe("Disabled exampled", () => {
  accountsData.forEach((accountData) => {
    if (!accountData.disabled) {
      it(`create account no. ${accountData.row + 1}`, () => {
        // test code
      });
    } else {
      it.skip(`${accountData.disabledReason} - DDT Test`, () => {
        // Tady nic nedelame, jenom vypisujeme, ze test je disabled
      });
    }
  });
});
```


A výsledek v Cypress
![image](https://github.com/deni-adam/cypress-bank-app/assets/95618620/d2395c6c-7cb4-4d9b-80e4-1c42aac996c8)